rs/boundary_node/rate_limits/canister_client/README.md

```diff
@@ -0,0 +1,8 @@
+# Rate-Limiting Canister Client
+
+```
+cargo run -- \
```

Comment:
Shouldn't we standardize on Bazel here?

Reply:
Good point: I just replaced it with the name of the binary.